packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js

```diff
@@ -60,25 +60,27 @@ export default function InputDatePicker(props) {
 		),
 	].filter(Boolean);
 	return (
-		<Date.Manager
-			value={props.value}
-			textInput={props.textInput}
-			dateFormat={props.dateFormat}
-			onChange={(...args) => handlers.onChange(...args, inputRef.current)}
-			useUTC={props.useUTC}
-		>
-			<FocusManager
-				divRef={containerRef}
-				onClick={handlers.onClick}
-				onFocusIn={handlers.onFocus}
-				onFocusOut={handlers.onBlur}
-				onKeyDown={event => {
-					handlers.onKeyDown(event, inputRef.current);
-				}}
+		<div className="date-picker">
```

Comment:
Why did you add this extra div ?
I see that in another component you add a margin-right, I don't understand why

Reply:
I need this div with class "date-picker", so I can apply margin-right. the margin-right is for the space between date input and time input.